public/kibi_timeline.js

```diff
@@ -80,6 +81,18 @@ define(function (require) {
         if (!timeline) {
           // create a new one
           timeline = new vis.Timeline($element[0]);
+          var timezone = config.get('dateFormat:tz');
+          if (timezone !== 'Browser') {
+            var utcOffset = null;
+            if (timezone.startsWith('Etc/GMT')) {
+              utcOffset = timezone.charAt(7) + ('0' + timezone.substring(8)).slice(-2) + ':00';
```

Comment:
Cool idea ;o) But `slice` creates a new string... I'd go with https://github.com/sirensolutions/kibi_timeline_vis/pull/43#discussion_r77642285, what do you think ?